dotCMS/src/curl-test/ContentResourceV1.postman_collection.json

```diff
@@ -308,6 +308,1040 @@
 				}
 			},
```

Comment:
I'll include a test to pull self-related content (from both sides, parent and child). Probably, these cases are easier to test in an ITs, the code would be more legible and easy to maintain. The postman test is good for testing http responses for example, but testing heavier logic is by far easier in an IT.